packages/terra-clinical-label-value-view/src/LabelValueView.jsx

```diff
@@ -73,6 +78,14 @@ const LabelValueView = ({
     );
   }
 
+  if (topicTextId) {
```

Comment:
Can we add Jest test for new prop to ensure aria-labelledBy is rendered only when `topicTextId` is provided with value.

Reply:
I've added 4 new jest tests for both the new prop `topicTextId` and the prop I've added in my previous Jira 'isChildOfDescriptionList'. Thank you!